src/pages/DashboardPage.tsx

```diff
@@ -1,18 +1,46 @@
 import "./DashboardPage.css";
 import DashboardStats from "@/components/dashboard/statistics/DashboardStats";
 import Sidebar from "@/components/dashboard/sidebar/Sidebar";
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import QuestionList from "@/components/dashboard/question-list/QuestionList";
+import { Question } from "@/models/question.model";
+import { fallbackQuestions } from "@/models/fallback-data/questions.fallback";
```

Comment:
Would be better to remove this import and obtain the fallbackQuestions from FallbackQuestionRepository.getQuestions instead.